careers/jobs/fullstack-node-engineer.md

```diff
@@ -0,0 +1,99 @@
+
+## Key Responsibilities of the Role
+
+- Be hands-on in new and existing code bases
+- Be a team-player, contributing to discussions around code quality and best practice
+- Be involved with the entire delivery of a solution, from ideation to go-live
+- Work with client’s engineering teams; explaining and advising where you can
+
+## Day to Day Tasks
+
+- Develop new and maintain existing systems
+- Write supporting documentation for systems you produce
+- Write unit integration tests as a first-class citizen
+- Perform code reviews and participate actively in discussions
+
+## About You
+
+Please see our [Developer Proficiency](../../handbook/professional-development/developer-proficiency.md) for a full list of what we think makes a "Senior" developer, but some key call-outs are:
+
+- Clearly able to identify and follow predefined patterns or approaches in an existing code base
+- Most comfortable working on clearly defined, well scoped features or problems
+- Typically 2-5 years focused on a specific domain in a professional environment
+- Being able to decide the priority level of a bug and set of the appropriate course of action
+- Being able to understand user stories and translate them into technical requirements
+- Being able to keep the client up to date on the day to day progress of a project
+- Are ambitious and excited to grow with DVELP
+- Must be willing to travel to client locations when required
+
+### Technical Skills
+
+While we understand that good programmers can learn new skills, this role is ideally for someone who has good working knowledge and experience in the following:
+
+- React
+- Redux
+- Typescript
+- Node (Express)
+- GraphQL
+
+More generally, desirable but not required are:
+
+- Experience with Twilio APIs
+- Experience with Google AI APIs
+
+We know there's a lot more to programming than what's listed above - but we'd rather learn through talking where your strengths are, than prescribe a pre-set list of hard requirements.
+
+## Salary
+
+We've chosen not to disclose the sort of salary we're looking to pay, because everyone is individual and we want to give everyone what we think is very fair compensation. We want to have that conversation with you to ensure everyone's happy, rather than sticking to a number on a page.
+
+## Benefits
+
+There are a wide array of perks available to you as a DVELP employee - talk to us to find out more. We're also open to having conversations around what you may need that's specific to you - don't be afraid to ask.
+
+## Pension
+
+We offer a competitive private pension scheme - talk to us to find out more.
+
+## Location
+
+DVELP is a remote-first company, so working from home or a co-working space is your day-to-day. DVELP (and its parent group Sabio) do have offices around the UK however, so working from those is more than possible too. DVELP has large clients all around Europe, so occasional travel to their offices may be required.
```

Comment:
I assume we haven't decided on the EU only bit yet?

Reply:
Aye, until we know otherwise I'm going to leave like this.